content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How Application Security works in Datadog
+kind: guide
+---
+
+## Overview
+
+Datadog Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities.
+
+APM records information about each HTTP request, these are referred to as traces. Datadog Application Security uses the information APM is already collecting, and flags any attack attempts based on suspicious requests as security signals. Depending on your security signal settings, you can receive notifications from Slack, email, or PagerDuty.
```

Comment:
> Datadog Application Security uses the information APM is already collecting, and flags any attack attempts based on suspicious requests as security signals

Still not accurate as a Security Signals is an aggregation of suspicious requests.

A Suspicious requests is a trace where security activity - today attack attempts - have been flagged.